src/scripts/content/gitea.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+/* global togglbutton, $ */
+
+togglbutton.render('.time-desc:not(.toggl)',
+  { observe: true },
+  $container => {
+    const link = togglbutton.createTimerLink({
+      className: 'gitea',
+      description: descriptionSelector,
+      projectName: projectSelector,
+    });
+    $container.appendChild(link);
+  }
+);
+
+function descriptionSelector () {
+  const description = document.getElementById('issue-title');
+  const issueId = description.previousElementSibling;
+  return `${issueId.textContent} ${description.textContent}`;
+}
+
+function projectSelector () {
+  const $project = document.getElementsByClassName('repo-title')[0].children[3];
+  return $project ? $project.textContent.trim() : '';
+}
+
+function tagsSelector () {
```

Comment:
It looks like you forgot to add this function to one of the params when creating the button link.

Reply:
I have actually developed and tested the tag-parsing capability as you have noticed, but I've found that no matter how I pass the extracted tags to the button, it never shows up in the selector.

I suspected that the tags maybe need to exist in advance but since I couldn't test it I just didn't wire it in. If you think that this should work as is (passing the tags as an array of string) I can push another commit in no time.

Reply:
Indeed the code doesn't seem to support tags when creating entries using the `createTimerLink` function. But it does sound like a bug. I think you can pass the `tagSelector` as one of the params since you already have it done, and when the code gets fixed you'll have the tags working as a side effect.